runtime/sema/authaccount_type.go

```diff
@@ -192,11 +192,23 @@ var AuthAccountType = &SimpleType{
 					)
 				},
 			},
+			"keys": {
+				Kind: common.DeclarationKindField,
+				Resolve: func(identifier string, _ ast.Range, _ func(error)) *Member {
+					return NewPublicConstantFieldMember(
+						t,
+						identifier,
+						AuthAccountKeysType,
+						accountTypeKeysFieldDocString,
+					)
+				},
+			},
 		}
 	},
 	NestedTypes: func() *StringTypeOrderedMap {
 		nestedTypes := NewStringTypeOrderedMap()
 		nestedTypes.Set("Contracts", AuthAccountContractsType)
```

Comment:
Good idea to define a constant for the type name below. 
Could you please also add a constant for this name?

Reply:
Migrated these in a separate PR: https://github.com/onflow/cadence/pull/668/files#diff-b5688033a56b955357a3fc86e260200fe8e341d094cfed5f9bf870cc0ee2f87aR25-R40